wagtailio/static/sass/abstracts/_variables.scss

```diff
@@ -25,3 +25,9 @@ $breakpoints: ('medium' '(min-width: 599px)', 'large' '(min-width: 1023px)');
 // Site grid
 $site-width: 1440px;
 $gutter: 30px;
+
+// Border Radius
+$rounded-sm: 3px;
+$rounded-md: 6px;
+$rounded-lg: 10px;
+$rounded-xl: 15px;
```

Comment:
thanks for adding these! : )

Reply:
Yass no problem! I'll make a follow up PR after that is for the purpose of replacing where these values are used on other templates too!